docs/config/io_helidon_openapi_OpenApiUi.adoc

```diff
@@ -0,0 +1,57 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+ifndef::rootdir[:rootdir: {docdir}/..]
+:description: Configuration of io.helidon.openapi.OpenApiUi
+:keywords: helidon, config, io.helidon.openapi.OpenApiUi
+:basic-table-intro: The table below lists the configuration keys that configure io.helidon.openapi.OpenApiUi
+include::{rootdir}/includes/attributes.adoc[]
+
+= OpenApiUi (openapi) Configuration
```

Comment:
OpenAPI UI

Reply:
I have globally replaced "U/I" with "UI". 

There are some places where interface or class names (such as `OpenApiUi`) will remain but such cases should already be enclosed in single tick marks. 

This particular file is generated by a tool. Probably the tool should enclose the class/interface name in single ticks as it generates the file.